xrmocap/ops/bottom_up_association/bottom_up_associator.py

```diff
@@ -0,0 +1,169 @@
+# yapf: disable
```

Comment:
The folder is called `bottom_up_association`, and the `.py` in it should be called the specific name, for example `four_dimensional_association`?

Reply:
I will adopt it